package.json

```diff
@@ -6,47 +6,47 @@
   "license": "MIT",
   "private": true,
   "devDependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/plugin-external-helpers": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.3.4",
-    "@babel/plugin-transform-runtime": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.3.4",
-    "@fortawesome/fontawesome-free": "^5.7.2",
-    "babel-loader": "^8.0.0",
-    "bulma-pageloader": "^2.1.0",
-    "bulma-tooltip": "^2.0.2",
-    "clipboard": "^2.0.4",
-    "copy-webpack-plugin": "^5.0.0",
-    "core-js": "^2.6.5",
-    "css-loader": "^2.1.0",
+    "@babel/core": "^7.11.6",
+    "@babel/plugin-external-helpers": "^7.10.4",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
+    "@creativebulma/bulma-tooltip": "^1.2.0",
```

Comment:
Hello @Booksbaum ,

sorry for the delay between your PR and my review.

Can we please revert this change to require `bulma-tooltip >= 3.0.0` ?

This is the officially supported version of Fulma.Extensions.Wikiki.Tooltip. In general, I didn't upgrade the support to @creativebulma because the author kind of expected it's user to consume the library with extra JavaScript like for the datepicker. And I prefer pure SCSS dependencies which are easier to map and also you can use them however you want.

https://github.com/Fulma/Fulma/blob/master/src/Fulma.Extensions.Wikiki.Tooltip/Fulma.Extensions.Wikiki.Tooltip.fsproj#L24-L26